### PR TITLE
Capacity/closing

### PR DESCRIPTION
**Pull request by thermokarst**
- Drop workshop closing date
- Drop 'past workshops' from workshop list
- Track capacity per rate

### CODE REVIEW

**Comment by jakereps** [comment]
👍 This looks good to me, pending the overall Workshop cap talked about offline it should be ready for merge.

File: payments/views.py [hunk 1]
```diff
@@ -29,8 +28,7 @@ def get(self, request, *args, **kwargs):
 
 
 class WorkshopList(ListView):
-    queryset = Workshop.objects.filter(start_date__gte=timezone.now()) \
-        .filter(draft=False)
+    queryset = Workshop.objects.filter(sales_open=False)
```
Comment (jakereps) [inline payments/views.py]: `sales_open=True` and it'll be good to go!

Comment (thermokarst) [inline payments/views.py]: Gah, silly mistake

**Comment by jakereps** [comment]
Thanks @thermokarst, looks good to go 👍 !

**Pull request merged**